qa/suites/fs/upgrade/upgraded_client/from_nautilus/tasks/1-client-upgrade.yaml

```diff
@@ -4,4 +4,6 @@ tasks:
 - print: "**** done install.upgrade on client.0"
 - ceph-fuse:
     client.0:
+- kclient:
+    client.1:
```

Comment:
there is no client.1 in this suite. I think you want something like:

```
$ git diff --cached
diff --git a/qa/suites/fs/upgrade/upgraded_client/from_nautilus/tasks/1-client-upgrade.yaml b/qa/suites/fs/upgrade/upgraded_client/from_nautilus/tasks/1-client-upgrade.yaml
index 8e9a80441297..afbf0e11a17d 100644
--- a/qa/suites/fs/upgrade/upgraded_client/from_nautilus/tasks/1-client-upgrade.yaml
+++ b/qa/suites/fs/upgrade/upgraded_client/from_nautilus/tasks/1-client-upgrade.yaml
@@ -2,8 +2,3 @@ tasks:
 - install.upgrade:
     client.0:
 - print: "**** done install.upgrade on client.0"
-- ceph-fuse:
-    client.0:
-- kclient:
-    client.1:
-- print: "**** done remount client"
diff --git a/qa/suites/fs/upgrade/upgraded_client/from_nautilus/tasks/2-client-mount b/qa/suites/fs/upgrade/upgraded_client/from_nautilus/tasks/2-client-mount
new file mode 120000
index 000000000000..e3600f453f2f
--- /dev/null
+++ b/qa/suites/fs/upgrade/upgraded_client/from_nautilus/tasks/2-client-mount
@@ -0,0 +1 @@
+.qa/cephfs/mount/
\ No newline at end of file
diff --git a/qa/suites/fs/upgrade/upgraded_client/from_nautilus/tasks/2-client-sanity.yaml b/qa/suites/fs/upgrade/upgraded_client/from_nautilus/tasks/3-client-sanity.yaml
similarity index 100%
rename from qa/suites/fs/upgrade/upgraded_client/from_nautilus/tasks/2-client-sanity.yaml
rename to qa/suites/fs/upgrade/upgraded_client/from_nautilus/tasks/3-client-sanity.yaml
```

Reply:
@batrick Sorry I think I missed your comment.
Yeah, I forgot that there only one client. I meant to `s/1-mds-1-client-micro.yaml/1-mds-2-client-micro.yaml/`. But yours patch looks better.

I will update it. Thanks

Reply:
@batrick This has been fixed, could you review it again ? This `requested changes` blocks it to get merged. Thanks.